csharp/AppEncryption/AppEncryption.IntegrationTests/Regression/CrossPartitionDecryptTest.cs

```diff
@@ -1,43 +0,0 @@
-using GoDaddy.Asherah.AppEncryption.Exceptions;
```

Comment:
We should hang on to this regression test.

Reply:
Huh strange. This must have been a part of that commit revert. I'll double check on this.